pkg/thanos/rules.go

```diff
@@ -207,7 +201,7 @@ func (o *Operator) selectRuleNamespaces(p *monitoringv1.ThanosRuler) ([]string,
 func makeRulesConfigMaps(t *monitoringv1.ThanosRuler, ruleFiles map[string]string, opts ...operator.ObjectOption) ([]v1.ConfigMap, error) {
 	//check if none of the rule files is too large for a single ConfigMap
 	for filename, file := range ruleFiles {
-		if len(file) > maxConfigMapDataSize {
+		if len(file) > operator.MaxConfigMapDataSize {
```

Comment:
same remark than before: `promRuleSelector.Select()` will prevent "too big" rules from reaching here so we can remove this check.